packages/data-context/src/sources/ProjectDataSource.ts

```diff
@@ -11,6 +11,7 @@ import { defaultSpecPattern } from '@packages/config'
 import parseGlob from 'parse-glob'
 import micromatch from 'micromatch'
 import RandExp from 'randexp'
+import fs from 'fs'
```

Comment:
Should this be `fs-extra`? I think we prefer this over `fs` everywhere. We should lint against this, probably.